### DISCUSSION TITLE
--url option is mandatory

### DISCUSSION THREAD

**Discussion by Nerwen18** [jfrog/jfrog-cli#2647] [General]
Hello everyone. I'm using jf inside my pipelines for almost 2 years, but lately, I get the following error on a daily basis: "the --url option is mandatory". I've tried to run jfrog rt ping command, and it requested the JFrog platform URL. Is there any way to get a platform URL without account, or any way to solve this issue?
Thanks in advance for any suggestion!

**Comment by sverdlov93** [comment]
Hi @Nerwen18 ,thanks for reaching out!
 jf rt ping requires JFrog Artifactory URL.
` --url                     [Optional] JFrog Artifactory URL. (example: https://acme.jfrog.io/artifactory)`
It also, supports using the default server config configured by `jf c add `or a specific one by setting `--server-id=` flag
Can  you provide your full use case an logs?

Other jfrog cli commands may require JFrog platform URL according to the --help section.
